src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/SimpleCountCollection.java

```diff
@@ -86,6 +124,51 @@ private static SimpleCountCollection readHDF5(final HDF5File file) {
         return new SimpleCountCollection(metadata, simpleCounts);
     }
 
+    /**
+     * Read all counts from a Google Cloud Storage URL.
+     * A corresponding index for the counts file must also be present.
+     */
+    public static SimpleCountCollection readFromGCS(final String path) {
+        IOUtils.assertFileIsReadable(IOUtils.getPath(path));
+        Utils.validate(BucketUtils.isCloudStorageUrl(path), "Read-count path must be a Google Cloud Storage URL.");
+        Utils.validate(new SimpleCountCodec().canDecode(path), String.format(
+                "Read-count file extension must be one of the following: [%s]",
+                String.join(",", SimpleCountCodec.SIMPLE_COUNT_CODEC_EXTENSIONS)));
```

Comment:
OK, no action.  I'm a little hesitant since `.tsv` is such a common file extension, and I would want to make sure that both the code and formats don't ever get to a place where it's easy to fail silently.